livekit-agents/livekit/agents/voice/room_io/room_io.py

```diff
@@ -59,9 +64,11 @@ class RoomInputOptions:
     audio_num_channels: int = 1
     noise_cancellation: rtc.NoiseCancellationOptions | None = None
     text_input_cb: TextInputCallback = _default_text_input_cb
-    sync_transcription: NotGivenOr[bool] = NOT_GIVEN
-    """False to disable transcription synchronization with audio output.
-    Otherwise, transcription is emitted as quickly as available."""
+    participant_kinds: NotGivenOr[list[rtc.ParticipantKind.ValueType]] = NOT_GIVEN
+    """Participant kinds accepted for auto subscription. If empty, all kinds are accepted."""
```

Comment:
```suggestion
    """Participant kinds accepted for auto subscription. If empty, all kinds are accepted."""
```
I guess we use DEFAULT_PARTICIPANT_KINDS so not all?

Reply:
updated